src/jvm/test/resources/expected_logs/spin_lock_in_incorrect_results_failure.txt

```diff
@@ -0,0 +1,38 @@
+= Invalid execution results =
+Parallel part:
+| a(): void       | c(): void       |
+| b(): void [1,0] | d(): 0    [2,1] |
+
+---
+Values in "[..]" brackets indicate the number of completed operations
+in each of the parallel threads seen at the beginning of the current operation
+---
+
+= The following interleaving leads to the error =
+Parallel part trace:
+|                      | c()                                                                                                 |
+|                      |   /* The following events repeat infinitely until thread switch */                                  |
```

Comment:
Let's replace with "// The following events until the thread switch repeat infinitely"